src/main/java/de/caritas/cob/uploadservice/config/TenantServiceApiClientConfig.java

```diff
@@ -0,0 +1,30 @@
+package de.caritas.cob.uploadservice.config;
+
+import de.caritas.cob.uploadservice.tenantservice.generated.ApiClient;
+import de.caritas.cob.uploadservice.tenantservice.generated.web.TenantControllerApi;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+@Component
+public class TenantServiceApiClientConfig {
```

Comment:
Could we initialize the api client conditional based on existing "multiTenancy" property? I want to prevent the need to set a api url for tenantservice even if the feature is disabled

Reply:
done